routes.js

```diff
@@ -1,58 +1,76 @@
-const { error } = require('console');
 const express = require('express');
-const router = express.Router();
+const createMyCustomApi = express.Router();
```

Comment:
To stay consistent with naming approach described before, please pick a better name for this router object. Something towards `membersApi` or `membersRoute`

Reply:
renamed